assisted-events-scrap/events_scrap/assisted_service_api.py

```diff
@@ -107,11 +106,12 @@ def create_cluster(
     ) -> models.cluster.Cluster:
         cluster = models.ClusterCreateParams(name=name, ssh_public_key=ssh_public_key, **cluster_params)
         log.info("Creating cluster with params %s", cluster.__dict__)
-        result = self.client.register_cluster(new_cluster_params=cluster)
+        result = self.client.v2_register_cluster(new_cluster_params=cluster)
         return result
 
-    def create_infra_env(self, name: str, ssh_public_key: Optional[str] = None, **infra_env_params
-                         ) -> models.infra_env.InfraEnv:
+    def create_infra_env(
+            self, name: str, ssh_public_key: Optional[str] = None, **infra_env_params
+    ) -> models.infra_env.InfraEnv:
```

Comment:
why in the first place we need the ability to create infra-envs here?
iiuc it should be a simple service that accesses just one endpoint for gathering events

Reply:
This is a larger question that not related to this PR. We can discuss it offline